tracing/src/macros.rs

```diff
@@ -801,6 +801,48 @@ macro_rules! event {
     );
 }
 
+/// The same as [`enabled!`], but queries subscribers specifically for an event,
+/// whereas [`enabled!`] queries for an event _or_ span.
```

Comment:
IMO, it's better to always have the documentation start with a summary of what the documented item does, and _then_ say "this is similar to $WHATEVER", instead of just leading with "this is the same as $WHATEVER". maybe something like:
```suggestion
/// Tests whether an event with the specified level and target would be enabled.
///
/// This is similar to [`enabled!`], but queries the current collector specifically for
/// an event, whereas [`enabled!`] queries for an event _or_ span.
///
/// See the documentation for [`enabled!]` for more details on using this macro.
```